bom/pom.xml

```diff
@@ -99,6 +99,105 @@ quarkus.vertx.prefer-native-transport=true
+    <fixed.auth.amqp.port>5674</fixed.auth.amqp.port> <!-- Reserved - not used -->
+    <fixed.deviceregistry.amqps.port>5675</fixed.deviceregistry.amqps.port> <!-- Reserved - not used -->
+    <fixed.deviceregistry.amqp.port>5676</fixed.deviceregistry.amqp.port>
+    <fixed.deviceregistry.http.port>1080</fixed.deviceregistry.http.port>
+    <fixed.deviceregistry.https.port>1443</fixed.deviceregistry.https.port> <!-- Reserved - not used -->
+    <fixed.commandrouter.amqps.port>5677</fixed.commandrouter.amqps.port> <!-- Reserved - not used -->
+    <fixed.commandrouter.amqp.port>5678</fixed.commandrouter.amqp.port>
+
+    <fixed.http.adapter.port>8080</fixed.http.adapter.port>
+    <fixed.https.adapter.port>8443</fixed.https.adapter.port>
+    <fixed.mqtt.adapter.port>1883</fixed.mqtt.adapter.port>
+    <fixed.mqtts.adapter.port>8883</fixed.mqtts.adapter.port>
+    <fixed.amqp.adapter.port>5672</fixed.amqp.adapter.port>
+    <fixed.amqps.adapter.port>5671</fixed.amqps.adapter.port>
+    <fixed.coap.adapter.port>5683</fixed.coap.adapter.port>
+    <fixed.coaps.adapter.port>5684</fixed.coaps.adapter.port>
+    <fixed.lora.adapter.port>2080</fixed.lora.adapter.port>
+    <fixed.loras.adapter.port>2443</fixed.loras.adapter.port>
+
+    <quarkus.application-dev.properties>
+hono.auth=${hono.auth}
+hono.auth.host=localhost
+hono.auth.port=${fixed.auth.amqps.port}
+hono.auth.name=${app.id}
+hono.auth.trustStorePath=../../demo-certs/certs/trusted-certs.pem
+hono.auth.validation.certPath=../../demo-certs/certs/auth-server-cert.pem
+hono.auth.jwksPollingInterval=PT20S
+
+hono.app.amqpMessagingDisabled=${hono.amqp-messaging.disabled:true}
+hono.app.kafkaMessagingDisabled=${hono.kafka-messaging.disabled:false}
+
+hono.kafka.commonClientConfig."bootstrap.servers"=${docker.host.address:localhost}:${kafka.port:9094}
+hono.kafka.commonClientConfig."max.block.ms"=1000
+
+# AMQP connection to central AMQP message broker if configured
+#hono.command.hostnameVerificationRequired=false
+hono.command.host=${docker.host.address:localhost}
+hono.command.port=${qpid.amqps.port:15671}
+hono.command.amqpHostname=hono-internal
+hono.command.keyPath=../../demo-certs/certs/${app.id}-key.pem
+hono.command.certPath=../../demo-certs/certs/${app.id}-cert.pem
+hono.command.trustStorePath=../../demo-certs/certs/trusted-certs.pem
+#hono.command.trustStorePassword=../../demo-certs/certs/trust-store-password
+hono.command.linkEstablishmentTimeout=${link.establishment.timeout:5000}
+hono.command.flowLatency=${flow.latency:1000}
+hono.command.requestTimeout=${request.timeout:500}
+
+hono.registration.host=localhost
+hono.registration.port=${fixed.deviceregistry.amqp.port}
+hono.registration.username=${app.id}@HONO
+hono.registration.password=${short.name}-secret
+hono.registration.linkEstablishmentTimeout=${link.establishment.timeout:5000}
+hono.registration.flowLatency=${flow.latency:1000}
+hono.registration.requestTimeout=${request.timeout:500}
+
+hono.tenant.host=localhost
```

Comment:
Does this mean that all adapters started locally expect the Device Registry to also have been started locally (and not on Docker)? I assume that when running the services on Docker and the adapters locally, you would need to override this value on the command line when starting the adapter using `mvn quarkus:dev`, right?